v22.1/architecture/reads-and-writes-overview.md

```diff
@@ -2,20 +2,20 @@
 title: Reads and Writes in CockroachDB
 summary: Learn how reads and writes are affected by the replicated and distributed nature of data in CockroachDB.
 toc: true
-docs_area: reference.architecture 
+docs_area: reference.architecture
 ---
 
-This page explains how reads and writes are affected by the replicated and distributed nature of data in CockroachDB. It starts by summarizing some important [CockroachDB architectural concepts](overview.html) and then walks you through a few simple read and write scenarios.
+This page explains how reads and writes are affected by the replicated and distributed nature of data in CockroachDB. It starts by summarizing how CockroachDB executes queries and then walks you through a few simple read and write scenarios.
 
 {{site.data.alerts.callout_info}}
 For a more detailed information about how transactions work in CockroachDB, see the [Transaction Layer](transaction-layer.html) documentation.
 {{site.data.alerts.end}}
 
-## Important concepts
+## Query execution
 
-{% include {{ page.version.version }}/misc/basic-terms.md %}
+See [CockroachDB architecture concepts](../glossary.html#cockroachdb-architecture) to review the CockroachDB features involved in query execution.
```

Comment:
Similar to my comment above, this is not really equivalent reader UX to what we had before. 

Any chance of using an include to keep these terms here?  that would let them still be maintained in one place, and it would still show up in the larger glossary